app/models/claim/verify_response_parameters_parser.rb

```diff
@@ -29,7 +29,7 @@ def initialize(response_parameters)
     end
 
     def attributes
-      identity_attributes.merge(govuk_verify_fields: govuk_verify_fields, verify_response: @response_parameters)
+      identity_attributes.merge(govuk_verify_fields: govuk_verify_fields)
```

Comment:
I think we need to update the Tomdoc for this model as well: https://github.com/DFE-Digital/dfe-teachers-payment-service/blob/870efe931ffadf2f32450be696d3c2690b5cded0/app/models/claim/verify_response_parameters_parser.rb#L18-L23